python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -1649,6 +1649,12 @@ def findArtifacts(self, templateCoadd, tempExpRefList, imageScalerList):
 
                 if self.config.doFilterMorphological:
                     maskName = self.config.streakMaskName
+                    # clear single frame streak mask if it exists already
+                    try:
```

Comment:
The faster your Apple Silicon CPU is the slower this gets in a loop.

Reply:
The problem is not the python exceptions, it's the c++ exceptions.  But now that I look at the afw code I don't know if we have an easy way of doing this without doing `getMaskPlaneDict()`.  I actually think this will be fine because it (a) is rare and (b) isn't per-source.

Reply:
Switched to check if its in `getMaskPlaneDict`